docs/monitors/cpu.md

```diff
@@ -0,0 +1,61 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# cpu
+
+
+This monitor reports cpu and cpu utilization metrics.
+
+```yaml
+monitors:
+ - type: cpu
+```
+
+
+Monitor Type: `cpu`
+
+[Monitor Source Code](https://github.com/signalfx/signalfx-agent/tree/master/internal/monitors/cpu)
+
+**Accepts Endpoints**: No
+
+**Multiple Instances Allowed**: **No**
+
+## Configuration
+
+| Config option | Required | Type | Description |
+| --- | --- | --- | --- |
+| `procFSPath` | no | `string` | The path to the proc filesystem. Useful to override in containerized environments.  (Does not apply to windows) (**default:** `/proc`) |
```

Comment:
We should probably also recognize an envvar for this, otherwise it will be very tedious to configure all these plugins with the procfs path.

Reply:
ack